src/CodeGeneration/ApiGenerator/Overrides/Descriptors/RefreshDescriptorOverrides.cs

```diff
@@ -0,0 +1,23 @@
+using ApiGenerator.Domain;
+
+namespace ApiGenerator.Overrides.Descriptors
```

Comment:
This can also be achieved by adding an `*.obsolete.json` companion file to the rest spec itself see:

https://github.com/elastic/elasticsearch-net/blob/100cf7c3f81553456fb039ca7a499a97c7990028/src/CodeGeneration/ApiGenerator/RestSpecification/Core/cat.thread_pool.obsolete.json